lib/doc/worksheet.js

```diff
@@ -867,8 +867,25 @@ class Worksheet {
     this._media = value.media.map(medium => new Image(this, medium));
     this.sheetProtection = value.sheetProtection;
     this.tables = value.tables.reduce((tables, table) => {
-      const t = new Table();
+      const t = new Table(this, null);
+      table.ref = table.tableRef;
+      table.headerRow = true;
+      table.rows = [];
       t.model = table;
+      // Get table rows
+      const {tl, br} = colCache.decodeEx(table.ref);
+      const height = br.row - tl.row + 1;
+      const width = br.col - tl.col + 1;
+      for (let i = 1; i < height; i++) {
+        const rowValues = [];
+        const row = this.getRow(tl.row + i);
+        for (let j = 0; j < width; j++) {
+          const cell = row.getCell(tl.col + j);
+          rowValues.push(cell.value);
+        }
+        t.addRow(rowValues);
+      }
+      t.commit();
```

Comment:
It may be better to move this code to the table.js as a named constructor, so this mapping is in the same place as where all other table mapping code is.